pbxplore/analysis/utils.py

```diff
@@ -60,7 +60,7 @@ def _slice_matrix(mat, idx_first_residue=1, residue_min=1, residue_max=None):
     # Take in account the optional offset (idx_first_residue)
 
     # range of indexes of the matrix
-    residues_idx = range(idx_first_residue, mat.shape[0] + idx_first_residue)
+    residues_idx = list(range(idx_first_residue, mat.shape[0] + idx_first_residue))
```

Comment:
Does this really need to be a list. I think it was used as an iterator when running on python 3.

Reply:
Indeed, 2to3 artifact, solved it